src/Rector/RemoveMethodCallRector.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare (strict_types=1);
+
+namespace Contao\Rector\Rector;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\Expression;
+use PHPStan\Type\ObjectType;
+use Rector\Contract\Rector\ConfigurableRectorInterface;
+use Rector\Rector\AbstractRector;
+use Webmozart\Assert\Assert;
+
+class RemoveMethodCallRector extends AbstractRector implements ConfigurableRectorInterface
+{
+    /**
+     * @var list<array{0: string, 1: string}>
+     */
+    private array $removedMethods = [];
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes() : array
+    {
+        return [Expression::class];
```

Comment:
As discussed, we probably want to check for a MethodCall if it's possible.

The provided example in rector/discussions/8376 is for removing the node, not replacing as we do in this case 🤔